Kudu.Core/Deployment/Generator/MicrosoftSiteBuilder.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Kudu.Contracts.Settings;
+using System;
+using System.Text;
+
+namespace Kudu.Core.Deployment.Generator
+{
+
+    public abstract class MicrosoftSiteBuilder : GeneratorSiteBuilder
+    {
+        private readonly string _projectFilePath;
+        private readonly string _solutionPath;
+        private readonly string _commandArgument;
+
+        protected string solutionPath
```

Comment:
Casing should be SolutionPath. Also, use automatic prop and get rid of _solutionPath:

    protected string SolutionPath { get; private set; }